nixos/tests/dynomite.nix

```diff
@@ -0,0 +1,35 @@
+import ./make-test.nix ({ pkgs, lib, ... }: {
+  name = "dynomite";
+  meta.maintainers = [ lib.maintainers.Scriptkiddi ];
+
+  machine =
+    { ... }:
+    {
+      virtualisation.memorySize = 1024;
+      environment.systemPackages = with pkgs; [ redis dynomite ];
+      services.dynomite = {
+        enable = true;
+        settings = {
+          dyn_o_mite = {
+            listen = "127.0.0.1:8102";
+            dyn_listen = "127.0.0.1:8101";
+            tokens = "101134286";
+            servers = [
+              "127.0.0.1:6379:1"
+              ];
+            data_store =  0;
+            mbuf_size = 16384;
+            max_msgs = 300000;
+          };
+        };
+      };
+    };
+
+  testScript = ''
+    startAll;
+    $machine->waitForUnit('redis.service');
+    $machine->waitForUnit('dynomite.service');
+    $machine->succeed('redis-cli -p 8102 ping') =~ /is valid/ or die;
+    $machine->succeed('dynomite --test-conf -c $(systemctl cat dynomite | grep ExecStart | cut -d"=" -f3) ') =~ /is valid/ or die;
```

Comment:
:thinking:

Reply:
Is there another way to access the filepath of a config file for a modul? I could not find any

Reply:
No, I guess not... I guess most modules just have their configuration file available to be referenced outside the module either through a `configFile` option, `/etc/module/config.file`, etc... OK.

Reply:
Could `dynamite --test-conf` be integrated into the `preStart` script of the service itself such that it fails with an invalid config immediately?